packages/template-retail-react-app/app/commerce-api/utils.test.js

```diff
@@ -19,6 +19,13 @@ const createJwt = (secondsToExp) => {
     return token.compact()
 }
 
+jest.mock('./utils', () => {
+    const originalModule = jest.requireActual('./utils')
+    return {
+        ...originalModule
+    }
+})
```

Comment:
same comment for https://github.com/SalesforceCommerceCloud/pwa-kit/pull/875/files#r1067567033